core/sql/regress/compGeneral/TEST071

```diff
@@ -4,6 +4,26 @@
 -- Simple test case to set up a table with computed col key prefix,
 -- do simple inserts, updates, deletes, merges.
 -------------------------------------------------------------------------
+-- @@@ START COPYRIGHT @@@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one
+-- or more contributor license agreements.  See the NOTICE file
+-- distributed with this work for additional information
+-- regarding copyright ownership.  The ASF licenses this file
+-- to you under the Apache License, Version 2.0 (the
+-- "License"); you may not use this file except in compliance
+-- with the License.  You may obtain a copy of the License at
+--
+--   http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing,
+-- software distributed under the License is distributed on an
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+-- KIND, either express or implied.  See the License for the
+-- specific language governing permissions and limitations
+-- under the License.
+--
+-- @@@ END COPYRIGHT @@@
 
```

Comment:
Note to reviewers: I spent some time trying to design a test case for these changes, but found it difficult due to the many CQDs we introduce in the Regress environment. So I decided to defer this to the next check-in. But while I was fiddling with test cases, I noticed this one test lacking a copyright.